### PR TITLE
New the case page

### PR DESCRIPTION
**Pull request by henwasaia**
1、To solve the problem ：Write a case page
2、solution ：case.html
3、test results : The page of the case was successfully written

### CODE REVIEW

File: themes/rainbond/layouts/partials/footer.html [hunk 1]
```diff
@@ -16,7 +16,8 @@
         </div>
         <div class="columns">
             <div class="column">
-                <p style="text-align: center;">© 2020 北京好雨科技有限公司 <a href="http://beian.miit.gov.cn/">京ICP备15028663号-4</a>
+                <p style="text-align: center;">© 2020 北京好雨科技有限公司 <a
```
Comment (barnettZQG) [inline themes/rainbond/layouts/partials/footer.html:19]: change "2020" to "2021"

File: themes/rainbond/layouts/partials/footer.html [hunk 2]
```diff
@@ -45,6 +46,8 @@
 <script type="text/javascript" src="/js/jquery.loading.min.js"></script>
 <script type="text/javascript" src="/js/clipboard.js"></script>
 <script type="text/javascript" src="/js/documentation.js?version={{.Site.Params.version}}"></script>
+<script type="text/javascript" src="/js/case.js?version={{.Site.Params.version}}"></script>
+<!-- <script type="text/javascript" src="/js/svg-inject.js"></script> -->
```
Comment (barnettZQG) [inline themes/rainbond/layouts/partials/footer.html:50]: remove comment code.

File: themes/rainbond/layouts/partials/header.html [hunk 3]
```diff
@@ -86,15 +96,22 @@
                 <a class="navbar-item is-secondary" href="https://store.goodrain.com" target="_blank">
                     Rainstore
                 </a>
-                <a class="navbar-item is-secondary "  href="https://cloud.goodrain.com/page/price" target="_blank">
+                <a class="navbar-item is-secondary " href="https://cloud.goodrain.com/page/price" target="_blank">
                     企业服务
                 </a>
             </div>
```
Comment (barnettZQG) [inline themes/rainbond/layouts/partials/header.html:102]: Lack of navigation:
```
<a href="/case/" class='navbar-item is-secondary {{if eq .active "case" }} is-active {{end}} ' target="_blank">案例 </a>
```

File: themes/rainbond/layouts/partials/footer.html (hunk 1 above)
Comment (henwasaia) [reply themes/rainbond/layouts/partials/footer.html]: Successfully changed

File: themes/rainbond/layouts/partials/footer.html (hunk 2 above)
Comment (henwasaia) [reply themes/rainbond/layouts/partials/footer.html]: Successfully remove comment code

File: themes/rainbond/layouts/partials/header.html (hunk 3 above)
Comment (henwasaia) [reply themes/rainbond/layouts/partials/header.html:105]: Added navigation

**Review by barnettZQG — APPROVED** [review]

**Pull request merged**